blackjack/console.py

```diff
@@ -0,0 +1,105 @@
+from core import *
+
+
+def show_cards(state: GameState):
+    print("")
+    print(f'Player 1: {render_hand(state.player_hand)}')
+    print(f'The House: {render_dealer(state.dealer_hand)}\n')
+
+
+def display_winner(winner: Win, state: GameState):
+    player_total = hand_total(state.player_hand)
+    house_total = hand_total(state.dealer_hand)
+    if winner is Win.Push:
+        print("")
+        print(f'The House: {render_hand(state.dealer_hand)}= {house_total}')
+        print(f'Player 1: {render_hand(state.player_hand)}= {player_total} \n PUSH!')
+        print("")
+    if winner is Win.Player_bj:
```

Comment:
If one of the previous if statements are caught, then there's no use in checking each one, right?

Reply:
Added returns to exit the function.